doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -120,7 +120,8 @@ Gazell
 Enhanced ShockBurst (ESB)
 -------------------------
 
-|no_changes_yet_note|
+* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+* Added support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
```

Comment:
```suggestion
* Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` and :ref:`zephyr:nrf54l15pdk_nrf54l15` boards.
```

Reply:
I've changet it to this entry:
```
  * Added:

    * Support for the :ref:`zephyr:nrf54l15pdk_nrf54l15` board.
    * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
```
Because two separate commits add entries to this file. Is this ok?

Reply:
It does not matter how many commits you have. One line is enough for these changes.

Reply:
I've added a one line entry.